### DISCUSSION TITLE
Database Functions revert to using double quotes?

### DISCUSSION THREAD

**Discussion by jimmiejackson414** [supabase/supabase#19151] [Questions] [answered]
I've got a naive Postgres question regarding functions. 

I'm working to create a seeder for my local environment using Snaplet. When it generates the seed.sql file based off of the snaplet config, I then try running `supabase db reset`, and receive the error "ERROR: column "My first notification" does not exist (SQLSTATE 42703)". This comes from a trigger I created where a default notification is created upon insert of a new user. That trigger looks like this:
```
  BEGIN
    INSERT INTO public.notifications (title, description, read_on_date, user_id)
    VALUES("My first notification", "This is my first notification.", null, new.id);
  
    RETURN NEW;
  END;
```
From what I understand, when creating triggers you can't use double quotes. So I replaced them with single quotes, the syntax colors returned, and I saved. When I try to run `supabase db reset` to apply the seeder though, it resets the triggers back to double quotes. 

What might I be missing here?

**Comment by GaryAustin1** [comment]
They are strings and have to be single quotes.  The error is because it is trying to find the name of something in the database because of the double quotes.   You might try casting 'my string'::text, but I don't know what is switching it to double quotes.

**Comment by jimmiejackson414** [comment, marked as answer]
Figured it out. I had three triggers that I combined into one to simplify, then had to go into the migration file that was generated to modify the triggers and functions manually to keep it from resetting to double quotes.
